Makefile

```diff
@@ -125,24 +121,12 @@ refresh-key: ## 🚀 Refresh a key on the instance
 	$(call check_defined, KMS_URL)
 	@CCF_PLATFORM=${CCF_PLATFORM};curl "${KMS_URL}"/app/refresh -X POST --cacert "${KEYS_DIR}"/service_cert.pem  -H "Content-Type: application/json" -i  -w '\n'
 
-set-constitution: start-host-idp ## Set new custom constitution
-	@echo -e "\e[34m$@\e[0m" || true
-	$(call check_defined, KMS_URL)
-	$(call check_defined, KEYS_DIR)
-	# Copy the files to the KEYS_DIR to construct the full constitution
-	if [ "${KMS_WORKSPACE}/sandbox_common" != "${KEYS_DIR}" ]; then \
-		echo "Copying files for constitution"; \
-		@sleep 5; \
-		cp -r ${KMS_WORKSPACE}/sandbox_common/*.js ${KEYS_DIR}; \
-	fi
-	@CCF_PLATFORM=${CCF_PLATFORM} ./scripts/submit_constitution.sh --network-url "${KMS_URL}" --certificate-dir "${KEYS_DIR}" --member-count ${MEMBER_COUNT}
-
 get-service-cert: # Get the mCCF service cert
 	@echo -e "\e[34m$@\e[0m" || true
 	$(call check_defined, IDENTITY_URL)
 	curl ${IDENTITY_URL} | jq ' .ledgerTlsCertificate' | xargs echo -e > ${KEYS_DIR}/service_cert.pem
 
-setup-mCCF: set-constitution deploy propose-add-key-release-policy propose-jwt-ms-validation-policy refresh-key  ## 🚀 Prepare an mCCF instance
+setup-mCCF: deploy propose-add-key-release-policy propose-jwt-ms-validation-policy refresh-key  ## 🚀 Prepare an mCCF instance
```

Comment:
Do we need this command at all now?

Reply:
Maybe, but I don't want to step on the toes of the other code in this PR more than is strictly necessary